components/Navbar/Navbar.tsx

```diff
@@ -299,6 +300,20 @@ export const Navbar = ({ active }: NavProps) => {
                 onClick={() => {
                   signOut(auth);
                   dispatch(setUserLoggedOut());
+                  mode
+                    ? toast.success('Logout was successful', {
+                        position: 'bottom-center',
+                        duration: 2000,
+                        style: {
+                          borderRadius: '10px',
+                          background: '#333',
+                          color: '#fff',
+                        },
+                      })
+                    : toast.success('Logout was successful', {
+                        position: 'bottom-center',
+                      });
+                  setMenuState(!menuState);
```

Comment:
what's the need for it? you are already putting it in the projects and about page